### DISCUSSION TITLE
I didn't find right document to install Optscale on Azure cloud, please help me getting steps

### DISCUSSION THREAD

**Discussion by ainapurapu** [hystax/optscale#340] [Q&A] [answered]
I didn't find the suitable document to install Optscale on Azure cloud; please help me get step by step approach

**Comment by stanfra** [comment, marked as answer]
Hi @ainapurapu, we do not have any limitations for Azure cloud,  you can deploy instance with our hardware requirements:
`The minimum hardware requirements for OptScale cluster: CPU: 8+ cores, RAM: 16Gb, SSD: 150+ Gb.
NVMe SSD is recommended.
OS Required: [Ubuntu 20.04](https://releases.ubuntu.com/focal/).
The current installation process does not work on Ubuntu 22.04`

and deploy cluster on it using README.md from optscale repo
